apps/web/src/components/Layout/Navbar.tsx

```diff
@@ -41,14 +45,23 @@ const Navbar: React.FC = () => {
             </Button>
           </Link>
         ) : null}
-        <div className="mr-auto">
-          <ProfileInfo
-            imageUrl={user.image}
-            name={user.name}
-            email={user.email}
-            id={user.id}
-          />
-        </div>
+
+        {!lg ? (
+          <div
+            onClick={toggleSidebar}
+            className="w-6 h-6 bg-natural-100 bg-opacity-50 rounded-[4px] p-[2px]"
+          >
+            <Menu />
+          </div>
+        ) : null}
```

Comment:
It is better to have it as a button. 

```suggestion
          <button
            type='button'
            onClick={toggleSidebar}
            className="w-6 h-6 bg-natural-100 bg-opacity-50 rounded-[4px] p-[2px]"
          >
            <Menu />
          </button>
        ) : null}
```